### ISSUE TITLE
Could it be possible to train image sequences?

### ISSUE THREAD

**Issue by ddrrnn123** [ajabri/videowalk#22]
Thank you so much for sharing the code! This idea is elegant and super cool! I am wondering whether this train.py can train the image sequences in each folder? I've tried to use both Imagefolder and the dataloader in test.py (Because the eval dataset is image sequence) to read image sequences. But both of them have the error with "XXX object does not support indexing". Or is there any method to transform the image sequences into videos that fit the training model?

Thank you very much!

**Comment by ajabri** [comment]
Yes, you just need to make a list in a text file with the following info:
```
/path/to/dir1 number_of_frames
/path/to/dir2 104
/path/to/dir3 134
/path/to/dir4 23
...
```
And then use it for argument `--data-path`.

**Comment by ddrrnn123** [comment]
> Yes, you just need to make a list in a text file with the following info:
> 
> ```
> /path/to/dir1 number_of_frames
> /path/to/dir2 104
> /path/to/dir3 134
> /path/to/dir4 23
> ...
> ```
> 
> And then use it for argument `--data-path`.

Thank you so much, Allan! It works well. Before your reply, I also modified the data loader vos.py from testing since it can load image sequences in testing. And I tried to use the bisect algorithm in VideoClips model to get the sequence clips with a specific frame_num and stride from different image sequences. I believe that is also a good option to organize the training data from image sequences.

Thanks again for your reply!
